spec/user_spec.rb

```diff
@@ -0,0 +1,43 @@
+require_relative "../lib/User"
+
+describe User do
+  let(:menu) {double(:menu,{starters: 
```

Comment:
Style/Blocks: Avoid using {...} for multi-line blocks.<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.